Sprint-2/implement/lookup.js

```diff
@@ -1,5 +1,11 @@
-function createLookup() {
+function createLookup(list) {
   // implementation here
+  let currency = {};
+  for(let i = 0; i < list.length; i++){
+    currency[list[i][0]] = list[i][1];
+  }
```

Comment:
This looks good!

There are a couple of other ways you could write this which - let's call the way you wrote 1, and I'll suggest 2 and 3 below. Can you respond to this comment saying what you prefer about each of the three approaches? I don't think any one of them is obviously the best :)

2:
```suggestion
  for(let item of list){
    currency[item[0]] = item[1];
  }
```

3:
```suggestion
  for(let [key, value] of list){
    currency[key] = value;
  }
```